content/src/ports/postgres.ts

```diff
@@ -52,6 +53,15 @@ export async function createDatabaseComponent(
   // Config
   const pool: Pool = new Pool(finalOptions)
 
+  /*
+  Please, don't remove this listener. From node-postgres documentation:
+  "It is important you add an event listener to the pool to catch errors.  Just like other event emitters,  if a pool
+  emits an error event and no listeners are added node will emit an uncaught error and potentially exit."
+  */
+  pool.on('error', (err) => {
```

Comment:
Which kind of errors are we actively ignoring with this lines?

Reply:
Connection-level errors. Running the integration tests locally, all the tests pass, but the process running them exists with error: 
`Error: Unhandled error. (error: terminating connection due to administrator command
    at Parser.parseErrorMessage (/Users/pedrotamborindeguy/repos/catalyst/node_modules/pg-protocol/src/parser.ts:369:69)...`
After some investigation, i found that the error is happening when the postgres container is shutting down in `jest.globalTeardown.ts`, AFTER the database component is stopped. 
I tried others workaround but i couldn't work it out.
I can remove this lines for this PR and continue working on it separately